src/ansys/fluent/core/solver/flobject.py

```diff
@@ -803,16 +810,38 @@ class FileName(Base):
     pass
 
 
+def expand_api_file_argument(command_name, value, kwargs):
+    """Expand API file argument."""
+    if kwargs.get("file_type") == "case-data" or command_name in [
```

Comment:
@hpohekar Let's avoid this command-specific code for now. Maybe we can decide some alternate strategy: transfer the file only if user has provided a relative path. Absolute paths will be sent to the server without any transfer/transformation. We can finalize it in next meeting/scrum. cc: @seanpearsonuk

Reply:
@mkundu1 This is based on my suggestion but this is supposed to be in a separate module. flobject.py would _try_ to import the function from that module. So, some other client that uses flobject.py can decide whether they need to implement it, with the idea being to preserve flobject's potential to become a framework. No specific command information would appear in this module.  It was appearing in this module in a different way prior to my proposal (please refer to origianl comment, which @hpohekar and I have discussed.

Reply:
Done. Thank you.